serverless.yml

```diff
@@ -48,6 +50,16 @@ custom:
     httpPort: ${env:API_PORT}
   pythonRequirements:
     layer: true
+  s3:
+    host: localhost
+    directory: /tmp
+
+resources:
+  Resources:
+    NewResource:
+      Type: AWS::S3::Bucket
+      Properties:
+        BucketName: ${env:TOKEN_METADATA_BUCKET}
```

Comment:
We have decided to create the bucket manually for now, so I think we don't need this, right?

Reply:
Removed